file-formats/aifi/examples/z_aff_example_aifi.aifi.json

```diff
@@ -0,0 +1,153 @@
+{
+  "formatVersion": "1",
+  "header": {
+    "description": "Example AIFD for ABAP File Format",
+    "originalLanguage": "en",
+    "abapLanguageVersion": "standard"
+  },
+  "generalInformation": {
+    "integrationType": "soap",
+    "soapSettings": {
+      "soapClass": "CL_SOAP_RFC_PROXY_TEST",
+      "soapMethod": "EXECUTE",
+      "soapRecordType": "/AIF/T100_SAP_FLIGHT"
+    },
+    "eventSettings": {
+      "repositoryId": "AIF_REPO_1",
+      "registrationId": "REGISTRATION_ID_EVT_GEN_1",
+      "registrationVersion": "5",
+      "topic": "Topic_0005"
+    },
+    "bgpfSettings": {
+      "operationId": "CL_AIF_TEST_BGPF_OPERATION",
+      "operationVariantId": "0000000001",
+      "operationVariant": "OP2023"
+    },
+    "namespace": "/AIF/",
+    "interfaceName": "IFTEST",
+    "interfaceVersion": "1",
```

Comment:
One more question for clarification: Is this representing the name and version of the current object or a reference to another interface?

If this is representing the name and version of this current object, it would be redundant as the filename represents the object name and the (aff) version is already contained in the `formatVersion`.

Reply:
This is AIF internal content which is stored in the object and the values are used by AIF framework only.

Reply:
So if the object name would be `Z_AFF_EXAMPLE_AIFI` in the `TADIR`, this `interfaceName` would have some other value than `Z_AFF_EXAMPLE_AIFI`?

Reply:
Correct

Reply:
Ok, thanks